include/Map/MapBase.hpp

```diff
@@ -20,6 +21,35 @@
 #define MAX_VIEWPOINTS 8
 #define MAX_MAP_UNK_130 0x20
 
+struct MapBase_Unk_13c {
+    /* 00 */ unk32 mUnk_00;
+    /* 04 */ unk32 mUnk_04;
+    /* 08 */ unk32 mUnk_08;
+};
```

Comment:
assuming you're sure this class ends there you can probably add the size, I tend to not doing that to indicate I'm unsure about it though so it's up to you
```suggestion
    /* 08 */ unk32 mUnk_08;
    /* 0c */
};
```